src/Event.php

```diff
@@ -514,6 +552,24 @@ public function setStacktrace(Stacktrace $stacktrace): void
         $this->stacktrace = $stacktrace;
     }
 
+    private static function getSdkPackages(): array
+    {
+        if (null === self::$sdkPackages) {
+            self::$sdkPackages = [];
+
+            foreach (Versions::VERSIONS as $package => $version) {
+                if (0 === strpos($package, 'sentry/')) {
```

Comment:
This is another nice side-effect: all Sentry official packages will be listed in the `sdk.packages` field of the event, with precise versions (up to tag or commit) and very little overhead.

Reply:
tbh I am not happy that this logic lives inside of the `Event`, the `Event` shouldn't know anything about packages or how/where to fetch them.
It should rather be a simple object holding data.

Also a side note the `sdk.packages` has no prio, we don't do anything with it yet. 
We discard it on the server right now.
While we probably want to have this at some point we don't need it now.

Reply:
Ok I'll discard it for now, we can still add it later. BTW it could be even moved inside the `EventFactory` to avoid logic inside the `Event`.